### PR TITLE
BREAKING CHANGE: bump all upgradable (dev)Dependencies except pure ESM

### PR DESCRIPTION
**Pull request by JounQin**
**What**:

Please view #437 for more details.

**Why**:

This PR is only a workaround to cut a major release for stupid `semantic-release`, see also https://github.com/prettier/prettier-eslint-cli/pull/438#issuecomment-1213593574

**How**:

N/A

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
->=12.22
```
Comment (kylemh) [inline .nvmrc:1]: how about `16.16.0`

**Review by kylemh — APPROVED** [review]

File: .nvmrc (hunk 1 above)
Comment (JounQin) [reply .nvmrc:1]: I have no opinion on this, all fine to me, personally. So I'll adopt this quickly.

File: .nvmrc [hunk 2]
```diff
@@ -1 +1 @@
...
+lts/*
```
Comment (JounQin) [inline .nvmrc:1]:
```suggestion
16.16.0
```

**Comment by idahogurl** [comment]
Nope. I suggest you try `sematic-release` locally on master before you do another PR.

```
12:10:06 AM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  The commit should not trigger a release
[12:10:06 AM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  Analyzing commit: feat!: bump all upgradable (dev)Dependencies except pure ESM (#437)
```

**Comment by JounQin** [comment]
https://github.com/prettier/prettier-eslint-cli/runs/7815418434?check_suite_focus=true#step:6:35

Stupid `sematic-release`, then I have no idea how to fix this. @idahogurl Any idea?

**Pull request merged**